app/Http/Controllers/JwtClaimsSupplier.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Resources\JwtClaims;
+use App\Models\User;
+
+class JwtClaimsSupplier extends Controller
+{
+    /**
+     * Return custom claims (for inclusion in JWT) for the given user
+     */
+    public function __invoke(User $user): JwtClaims
```

Comment:
I would also use conventional method names for controller. Since this returns a single object, then "show" appropriate here

Reply:
How are the privileges getting into the JWT? I'm assuming this returns all of the privileges for different institutions? This is unnecessary, JWT should contain only privileges for selected institution

Reply:
This was a very bad and stupid oversight of mine 🤦 . Thank you for your patience. I changed the endpoint’s input/output in the latest revision.